qa-tests-backend/src/main/groovy/util/Env.groovy

```diff
@@ -132,7 +136,7 @@ class Env {
 
             String password = null
             try {
-                def passwordPath = "../deploy/${envVars.get("CLUSTER").toLowerCase()}/central-deploy/password"
+                def passwordPath = "../deploy/${envVars.get("CLUSTER").toString().toLowerCase()}/central-deploy/password"
```

Comment:
Env vars returns `object` so it has nt `toLowerCase()` we need to convert it to string.